docs/pipelines/multisample/shiva.md

```diff
@@ -95,6 +96,28 @@ At this moment the following variant callers can be used
 | bcftools_singlesample | <a href="https://samtools.github.io/bcftools/bcftools.html">bcftools</a> |  |
 | varscan_cns_singlesample | <a href="http://varscan.sourceforge.net/">varscan</a> |  |
 
+##### Somatic
```

Comment:
A small section explaining why different variant callers must be used for somatic variant calling might not be a bad idea.